src/service/cid-impl.js

```diff
@@ -78,19 +86,24 @@ class Cid {
    *     The consent promise passed to other calls should then itself
    *     depend on the opt_persistenceConsent promise (and the actual
    *     consent, of course).
-   *  @return {!Promise<?string>} A client identifier that should be used
+   * @param {!CreateCookieParamsDef=} opt_createCookieParams If specified,
+   *     this parameter enables CID to create a cookie if it doesn't yet
+   *     exist.
+   * @return {!Promise<?string>} A client identifier that should be used
    *      within the current source origin and externalCidScope. Might be
    *      null if no identifier was found or could be made.
    *      This promise may take a long time to resolve if consent isn't
    *      given.
    */
-  get(externalCidScope, consent, opt_persistenceConsent) {
+  get(externalCidScope, consent, opt_persistenceConsent,
```

Comment:
I was thinking to maybe convert the first arg to a prop object. `opt_persistenceConsent` will be undefined for most callers (all but 1) while opt_createCookieParams should be set quite often